plugin/src/cli_utils/clusterinfo/mod.rs

```diff
@@ -0,0 +1,41 @@
+use plugin::ExecuteOperation;
+use crate::cli_utils::CliArgs;
+
+#[derive(clap::Subcommand, Debug)]
+pub(crate) enum Operations {
+    // Gets informations of all installed engines
+    Get(ClusterInfoArgs)
+}
+
+#[async_trait::async_trait(?Send)]
+impl ExecuteOperation for Operations {
+    type Args = CliArgs;
+    type Error = Error;
+
+    async fn execute(&self, _cli_args: &CliArgs) -> Result<(), Error> {
+        match self {
+            Operations::Get(cluster_info_args) => {
+                println!("Clusterinfo cmd reached, {}", cluster_info_args.chart)
```

Comment:
there's nothing here?
If there's no implementation yet you can add a todo!()

Reply:
Done